docs/arch-split.md

```diff
@@ -244,7 +261,7 @@ available unqualified until the end of the context block. Indeed, in this case,
 the only purpose of the anonymous context block is to limit the scope of this
 `interpretation`.
 
-Note: It is critical to the success of arch_split that we *never* interpret the
+Note: It is critical to the success of arch\_split that we *never* interpret the
```

Comment:
Should not need to escape underscore in markdown

Reply:
Shows up as an error in my editor. Maybe I should calm down my editor.

Reply:
Fixed